partiql-ast/src/main/kotlin/org/partiql/ast/Ast.kt

```diff
@@ -94,12 +94,12 @@ public object Ast {
     }
 
     @JvmStatic
-    public fun exprCall(function: IdentifierChain, args: List<Expr>, setq: SetQuantifier?): ExprCall {
+    public fun exprCall(function: IdentifierChain, args: List<Expr>, setq: SetQuantifier? = null): ExprCall {
```

Comment:
(self-review) this will allow for defaults from Kotlin code. What is our stance w/ providing Java defaults using `@JvmOverloads` annotation?

Alternative modeling could have not used defaults but provide two functions, though it can get unwieldy if there were multiple nullable arguments.

Reply:
From offline discussion,
- use `@JvmOverloads` when factory method does not have many nullable fields with defaults of `null`
- when factory method has many nullable fields (e.g. query set, SFW), specify common factory methods of required fields
  - also create the default nullable variant without the `@JvmOverloads`